client/blocks/login/login-form.jsx

```diff
@@ -80,6 +80,16 @@ export class LoginForm extends Component {
 	};
 
 	componentDidMount() {
+		if ( this.state.isFormDisabledWhileLoading ) {
+			if ( window && window.calypsoLoadStartTime && window.calypsoLoadStartTime[ 'log-in' ] ) {
+				this.props.recordTracksEvent( 'calypso_load_end', {
+					...window.calypsoLoadStartTime[ 'log-in' ],
+					elapsedMs: Date.now() - window.calypsoLoadStartTime[ 'log-in' ].startTimestamp,
+					path: '/log-in',
+				} );
+			}
```

Comment:
might want to unset `window.calypsoLoadStartTime[ 'log-in' ] )` here to prevent reporting this time more than once if the component remounts for some reason. Or flag that you've reported load time for login somewhere.